obspy/scripts/runtests.py

```diff
@@ -109,10 +109,19 @@ def main():
     os.chdir(base_obspy_path)
     try:
         status = pytest.main(plugins=[plugin])
+    except Exception as e:
```

Comment:
Catching the exception and reraising it. I think this has no effect.

Reply:
It was just required to use an `else` clause

Reply:
OK, I see, but here you are basically catching an exception inside pytest. I have not encountered such a case.

Reply:
Ya, it was required syntactically to have an else clause. I will just refactor though so we don't need either.